ui/components/component-library/button-icon/button-icon.scss

```diff
@@ -29,6 +29,10 @@
   }
 
   // ButtonIcon Sizes
+  &--size-xs {
+    --button-icon-size: 20px;
+  }
```

Comment:
Should this be removed? Started a discussion in slack regarding changes in https://github.com/MetaMask/metamask-extension/pull/24150

Reply:
Will address both comments in a follow-up PR in the merge train